backend/lib/event_queries.py

```diff
@@ -2,9 +2,9 @@
 import time
 from datetime import date
 from discord.ext import commands
-from game_queries import get_game_id
-from helper_commands import check_admin_status, AdminPermissionError
-from game_queries import GameNotFoundError
+from backend.lib.game_queries import get_game_id
+from backend.lib.helper_commands import check_admin_status, AdminPermissionError
+from backend.lib.game_queries import GameNotFoundError
```

Comment:
Import changes due to unit testing and running bot_controller.py from project base